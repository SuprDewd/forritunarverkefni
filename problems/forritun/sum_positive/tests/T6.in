571
809
-860
806
75
241
29
-997
28
494
