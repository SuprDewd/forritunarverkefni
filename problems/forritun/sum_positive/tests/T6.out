3053
8
2
